common/account_utils/src/validator_definitions.rs

```diff
@@ -339,19 +351,19 @@ pub fn is_voting_keystore(file_name: &str) -> bool {
         return true;
     }
 
-    // The format exported by the `eth2.0-deposit-cli` library.
+    // The format exported by the `eth2.definitions-deposit-cli` library.
```

Comment:
a few accidental `0 -> definitions` replacements here

Reply:
`%s/.0/.definitions/gI` *is* a bit aggressive. Noted.

Reply:
Consult d451101.